docs/src/main/sphinx/connector/pinot.rst

```diff
@@ -33,18 +33,36 @@ Configuration properties
-``pinot.controller-urls``      Yes        A comma separated list of controller hosts. If Pinot is deployed via
-                                          `Kubernetes <https://kubernetes.io/>`_ this needs to point to the controller
-                                          service endpoint. The Pinot broker and server must be accessible via DNS as
-                                          Pinot returns hostnames and not IP addresses.
-``pinot.segments-per-split``   No         The number of segments processed in a split. Setting this higher reduces the
-                                          number of requests made to Pinot. This is useful for smaller Pinot clusters.
-``pinot.request-timeout``      No         The timeout for Pinot requests. Increasing this can reduce timeouts if DNS
-                                          resolution is slow.
-============================== ========== ==============================================================================
+========================================================= ========== ==============================================================================
+Property Name                                             Required   Description
+========================================================= ========== ==============================================================================
+``pinot.controller-urls``                                 Yes        A comma separated list of controller hosts. If Pinot is deployed via
+                                                                     `Kubernetes <https://kubernetes.io/>`_ this needs to point to the controller
+                                                                     service endpoint. The Pinot broker and server must be accessible via DNS as
+                                                                     Pinot returns hostnames and not IP addresses.
+``pinot.segments-per-split``                              No         The number of segments processed in a split. Setting this higher reduces the
+                                                                     number of requests made to Pinot. This is useful for smaller Pinot clusters.
+``pinot.request-timeout``                                 No         The timeout for Pinot requests. Increasing this can reduce timeouts if DNS
+                                                                     resolution is slow.
+``pinot.authentication.controller.type``                  No         Pinot authentication method for controller requests. Allowed values are
+                                                                     ``NONE`` and ``BASIC_INLINE`` - defaults to ``NONE`` which is no
+                                                                     authentication.
+``pinot.authentication.controller.basic-inline.user``     No         Controller username for basic authentication method.
```

Comment:
I've now called this `basic-inline` because there could be more than one way to provide basic auth credentials, e.g. via password file